### ISSUE TITLE
Error: No provider for: jupyterlab-tour:ITourManager

### ISSUE THREAD

**Issue by tonimc** [jupyterlab-contrib/jupyterlab-tour#13]
Hi!

I'm trying to use this extension throw the `TourManager` but when I tried to ger the manager I see the following message on the console 

```
Error: No provider for: jupyterlab-tour:ITourManager.
    at W.e.resolveRequiredService (static/lab/jlab_core.20dcbc2dd05d71e94169.js?v=20dcbc2dd05d71e94169:2)
    at static/lab/jlab_core.20dcbc2dd05d71e94169.js?v=20dcbc2dd05d71e94169:2
    at Array.map (<anonymous>)
    at W.e.activatePlugin (static/lab/jlab_core.20dcbc2dd05d71e94169.js?v=20dcbc2dd05d71e94169:2)
    at static/lab/jlab_core.20dcbc2dd05d71e94169.js?v=20dcbc2dd05d71e94169:2
    at Array.map (<anonymous>)
```

Maybe my code is wrong but I'm doing the same I do in other extensions

```js
import { ITourManager } from 'jupyterlab-tour/lib/tokens';

const welcomeTour: JupyterFrontEndPlugin<void> = {
  id: `${PACKAGE_NAMESPACE}:welcomeTour`,
  autoStart: true,
  requires: [ITourManager],
  activate: (app: JupyterFrontEnd, tourManager: ITourManager) => {
  }
};
```

I'm trying to remove the default tours because I'm developing a custom integration of jupyterlab.

Anyone could say me what I should do?

**Comment by fcollonval** [comment]
Hey @tonimc thanks for reaching out.

You need to configure your package to ensure JupyterLab is using the shared package: https://jupyterlab.readthedocs.io/en/stable/extension/extension_dev.html#requiring-a-service. In this case: 

```json
"jupyterlab": {
  "sharedPackages": {
    "jupyterlab-tour": {
      "bundled": false,
      "singleton": true
     }
   }
 }
```

**Comment by tonimc** [comment]
I miss to add `bundled` on the `sharedPacakges`but it's not working for me too. 
Anyway I resolve the issue, at the end I get the `TourManager`class and I build my tours.

Thank you so much for your reply and for the extensions, it looks great!

**Comment by fcollonval** [comment]
@tonimc there was an error in the package that has been corrected in version 3.0.1. I tested it successfully in https://github.com/fcollonval/with_tour

**Comment by tonimc** [comment]
Thank you @fcollonval for your work!
